bor/README.md

```diff
@@ -5,8 +5,10 @@
 * [Preliminary terminology](#preliminary-terminology)
 * [Overview](#overview)
 * [How does it work](#how-does-it-work)
-* [How to propose a span](#how-to-propose-a-span)
+	* [How to propose a span](#how-to-propose-a-span)
```

Comment:
Why this space ?

Reply:
To make it a sub-section. This is also what @marcello33 [suggested](https://github.com/maticnetwork/heimdall/pull/1082#discussion_r1357763937) in the staking module README.